src/leappto/cli.py

```diff
@@ -780,8 +785,26 @@ def print_migrate_info(text):
             None
         )
 
-        return_code, claimed_names = mc.check_target()
-        for name in sorted(claimed_names):
+        check_result = {}
+        return_code, check_result['containers'] = mc.check_target_containers()
+
+        if parsed.status:
+            check_commands = dict()
+            check_commands['docker'] = 'docker info'
```

Comment:
I'm not sure this `docker` check is quite sufficient - due to the volume mount, we not only need the docker client to be installed, we also need the docker daemon to be running directly on the target system, rather than the client connecting to a daemon hosted elsewhere.